### ISSUE TITLE
Clean up Production database content, prior to upgrade 

### ISSUE THREAD

**Issue by daiClegg** [dbrownukk/EFD_v2#397]
Instance: EFD_OHEA
Project: all

Need to merge all the data into one project

**Comment by daiClegg** [comment]
@celiapetty I was able to re-create Okirio, 
Originally it was loaded into the 'NIMFRU (Lake Basin Fishing)' Project.
I replicated it in the 'NIMFRU (Lake Opeta/Bisina Basin Fishing)' Project.
I chose this as the target since it has the greatest amount of data already in it, so less work to move everything else there.
It took me less than 30 minutes to do it:

1. Created new Village & 3 WGs with all same properties as Okirio.
2. Generated spreadsheet templates for each WG (to set up the WGI records ready for upload)
3. Downloaded the 3 completed spreadsheets attached to the original Okirio WGs
4. Uploaded the relevant spreadsheet to each of the new Okirio WGs.
5. Parsed & Validated them. There are plenty of minor corrections to be made. The only ones of significance were Cash savings recorded as Non-Food Purchases. I moved them to Cash Assets.
6. Ran the Cash Income chart for the 2 Okirio Villages and compared
> Note the wild difference for the Middle WGs due to additional employment


The Employment tab on the MIddle WG spreadsheet (attached to both Okirio Middle WGs) has 1 entry.
But the validated record in the original project has 5 employments, which treble the cash income for the group. Where did that come from?
I have not examined whether further manual entires have been made.
And even if there are i have no way of judging whether they are valid.
